modules/@apostrophecms/piece-type/index.js

```diff
@@ -404,7 +405,7 @@ module.exports = {
           return self.apos.modules['@apostrophecms/job'].runBatch(
             req,
             req.body._ids,
-            async function(req, id) {
+            async function (req, id) {
```

Comment:
I see this space before opening paren both appearing and disappearing in this PR.

We don't normally have this space, please remove.

Reply:
Removed - somehow my linter is behaving badly. I need to dig into why.